game_board.py

```diff
@@ -0,0 +1,99 @@
+import pyglet
+
+
+class Cell:
+    def __init__(self, position_x, position_y, cell_size, state):
+        self.position_x = position_x    # position is bottom left corner
+        self.position_y = position_y
+        self.cell_size = cell_size
+        self.state = state
+
+    def add_cell_to_batch(self, batch):
+        left = self.position_x
+        right = self.position_x + self.cell_size
+        bottom = self.position_y
+        top = self.position_y + self.cell_size
+
+        vertex_list = batch.add(5, pyglet.gl.GL_LINE_STRIP, None,
+                                ('v2i', (left, bottom,
+                                         right, bottom,
+                                         right, top,
+                                         left, top,
+                                         left, bottom)),
+                                ('c3B', (0, 0, 255,
+                                         0, 0, 255,
+                                         0, 0, 255,
+                                         0, 0, 255,
+                                         0, 0, 255)))
+
+
+def draw_background(win_width, win_height,
+                    board_width, board_height, columns_num, rows_num):
```

Comment:
For similar reasons, the suggested style for situations like this one (a lot of arguments) would be:

```
def draw_background(
    win_width, win_height, board_width, board_height, columns_num, rows_num):
```

Reply:
changed